Skylab/Custom elements/CustomView.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  CustomView.swift
+//  Skylab
+//
+//  Created by Artem Tkachenko on 22.02.2023.
+//
+
+import UIKit
+
+@IBDesignable class CustomView: UIView {
```

Comment:
You need to think about the name of the component class. There are can be a few CustomViews in our project.

Reply:
I think we will all decide which name is best.

Reply:
I would propose BaseBackgroundView or MainBackgroundView or something like this :) 

But I fully agree with Serhiy, you to add some names during development, and if someone doesn't like your idea, he can leave a comment on your PR.